setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+from distutils.cmd import Command
+class TestCommand(Command):
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        import sys, subprocess
+
+        raise SystemExit(
+            subprocess.call(['nosetests',
```

Comment:
Got it. I believe there's also a generatable `runtests.py` which would be portable ... I'd like to add this to https://github.com/westurner/cookiecutter-saltformula ?